functions.js

```diff
@@ -0,0 +1,260 @@
+      <div class="mobile-pop-up-container-projects">
+        <ul class="mobile-pop-up-container-projects-element">
+          <li>Ruby on rails</li>
+          <li>css</li>
+          <li>Javascript</li>
+          <li>Html</li>
+        </ul>
+      </div>
+      <div class="mobile-pop-up-text">
+          <p>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea</p>
+      </div>
+      <div class="mobile-button-pop-up">
+        <button class="mobile-pop-up-container-btn" type="button" name="button">
+          <a href="./index.html">See Live
+        <i class="fab fa-github fa-sm modal-social"></i></a></button>
+        <button class="mobile-pop-up-container-btn" type="button" name="button">
+          <a href="./index.html">See Source
+        <i class="fab fa-github fa-sm modal-social"></i></a></button>
+    </div>`;
+  containerPopUp.innerHTML = content;
+};
+
+// to close the modal
+function closeModal() { // eslint-disable-line no-unused-vars
+  containerPopUp.style.display = 'none';
+}
+
+// This function validates the email on the formspree
+function ValidateEmail(inputText) {
+  const mailformat = '.*[A-Z]+.*';
+  if (!inputText.match(mailformat)) {
+    document.getElementById('email-fail-id').style.display = 'none';
+    return true;
+  }
+  document.getElementById('email-fail-id').style.display = 'flex';
+  return false;
+}
+
+function myFunction(e) { // eslint-disable-line no-unused-vars
+  if (!ValidateEmail(e.value)) {
+    document.querySelector('.contact-form-btn').setAttribute('disabled', 'true');
+  } else {
+    document.querySelector('.contact-form-btn').removeAttribute('disabled', 'true');
+  }
+}
+
+// To save form data in the local storage and retrive it on window load
+const formEl = document.querySelector('#contact-form');
+const userName = formEl.querySelector('#full-name');
+const userEmail = formEl.querySelector('#email');
+const userMesssage = formEl.querySelector('#message');
+
+const saveData = () => {
+  const formData = {
+    name: userName.valuee,
+    email: userEmail.value,
+    message: userMesssage.value,
+  };
+  localStorage.setItem('sessionData', JSON.stringify(formData));
+};
+
+userName.addEventListener('input', saveData);
+userEmail.addEventListener('input', saveData);
+userMesssage.addEventListener('input', saveData);
+
+window.addEventListener('load', () => {
+  const serializeFormData = JSON.parse(localStorage.getItem('sessionData'));
+  userName.value = serializeFormData.name;
+  userEmail.value = serializeFormData.email;
+  userMesssage.value = serializeFormData.message
```

Comment:
A semicolon is missing at the end of the statement. You might consider adding a trailing semicolon to fix this.